routes/alfresco.py

```diff
@@ -325,91 +396,150 @@ def run_aggregate_var_polygon(var_ep, poly_gdf, poly_id):
+                ds, poly_mask_arr, flammability_future_dim_encodings, bandname, varname
+            )
+    elif cov_id_str == "alfresco_vegetation_type_percentage":
+        ar5_results = summarize_within_poly_marr(
+            ds_list[-1], poly_mask_arr, veg_type_dim_encodings, bandname, varname
+        )
+        for era, summaries in ar5_results.items():
+            aggr_results[era] = summaries
+        aggr_results = remove_invalid_dim_combos(aggr_results)
 
     return aggr_results
 
 
+def remove_invalid_dim_combos(results):
+    """Remove data from invalid era/model/scenario dimension combinations
+
+    Args:
+        results (dict): point or area results data
+
+    Returns:
+        results (dict): point or area results data with invalid combos removed
+    """
+    eras = list(veg_type_dim_encodings["era"].values())
+    models = list(veg_type_dim_encodings["model"].values())
+    scenarios = list(veg_type_dim_encodings["scenario"].values())
+    # Remove empty data from invalid combos of era/model/scenario.
+    for era in eras:
+        for model in models:
+            # Remove non-CRU-TS models and non-historical era from historical data.
+            if era == "1950-2008":
+                if model != "CRU-TS":
+                    results[era].pop(model, None)
+                    continue
+                for scenario in scenarios:
+                    if scenario != "historical":
+                        results[era][model].pop(scenario, None)
+            # Remove historical era from projected data.
+            else:
+                results[era][model].pop("historical", None)
+        # Remove CRU-TS "model" from projected data.
+        if era != "1950-2008":
+            results[era].pop("CRU-TS", None)
+
+    return results
+
+
 def create_csv(data_pkg, var_ep, place_id=None, lat=None, lon=None):
     """Create CSV file with metadata string and location based filename.
     Args:
         data_pkg (dict): JSON-like object of data
-        var_ep (str): flammability or veg_change
+        var_ep (str): flammability, veg change, or veg_type
         place_id: place identifier (e.g., AK124)
         lat: latitude for points or None for polygons
         lon: longitude for points or None for polygons
     Returns:
         CSV response object
     """
     varname = var_ep_lu[var_ep]["varname"]
+    if var_ep in ["flammability", "veg_change"]:
+        fieldnames = flammability_fieldnames
+        stat = "mean"
+    elif var_ep == "veg_type":
+        fieldnames = veg_type_fieldnames
+        stat = "percent"
     csv_dicts = build_csv_dicts(
-        data_pkg,
-        alf_fieldnames,
-        {"variable": varname, "stat": "mean"},
+        data_pkg, fieldnames, {"variable": varname, "stat": stat},
     )
 
     place_name, place_type = place_name_and_type(place_id)
 
     metadata = csv_metadata(place_name, place_id, place_type, lat, lon)
 
-    if var_ep == "flammability":
+    if var_ep in "flammability":
```

Comment:
Is this right? Or should it be equivalence:
```suggestion
    if var_ep == "flammability":
```